desktop-widgets/mainwindow.cpp

```diff
@@ -36,6 +36,7 @@
 #include "qt-models/weigthsysteminfomodel.h"
```

Comment:
comment for the commit message of `67e6fc43`.
it's best to have a single short sentence with length around 70-80 characters as the commit title.
then the commit message body should start with a new sentence.

just a nitpick for the `...` ellipsis.